packages/model/src/calendar.ts

```diff
@@ -72,6 +74,8 @@ export interface CalendarResponse {
   id: string;
   name: string;
   color: GoogleLegacyCalendarColor;
+  activeGroups?: boolean;
+  incompleteWorkingGroups?: boolean;
```

Comment:
```suggestion
  hasIncompleteWorkingGroups?: boolean;
```

Reply:
I really don't like prefixing has on variables. I think its very easily inferred from the type and overly verbose. I think it also encourages good practices because if we ever needed to return an actual list of groups it wouldn't allow you to also maintain this flag with the same variable name

Reply:
i agree we should not prefix properties such as `active` with prefixes like `has` or `is`but i would actually use your example as a reason why we should do it here - if we ever want to return an actual list of `groups` or `activeGroups` we will not be able to as the name will already be taken. 

It is very confusing to have a `groups` property which is not an array of groups (like everywhere else) but a boolean. 

We want to be consistent and to leave the interfaces open for extension, not for modification (see what i did here)